Couchdb is disabled until ibm/java-sdk-core updates gson

### Bug description

Quarkus upgraded gson to 2.13.0 (see [PR](https://github.com/quarkusio/quarkus/commit/81d3813eda665ef8ef79ad4cbcf7fef3fc89ca22)),. This change causes `cauchdb` tests to fail

```	Caused by: java.lang.NoClassDefFoundError: com/google/gson/internal/$Gson$Types
		at com.ibm.cloud.sdk.core.util.DynamicModelTypeAdapterFactory.getBoundFields(DynamicModelTypeAdapterFactory.java:161)
		at com.ibm.cloud.sdk.core.util.DynamicModelTypeAdapterFactory.create(DynamicModelTypeAdapterFactory.java:108)
		at com.google.gson.Gson.getAdapter(Gson.java:628)
		at com.google.gson.Gson.toJson(Gson.java:928)
		at com.google.gson.Gson.toJson(Gson.java:899)
		at com.google.gson.Gson.toJson(Gson.java:848)
		at com.google.gson.Gson.toJson(Gson.java:825)
		at com.ibm.cloud.sdk.core.http.RequestBuilder.bodyContent(RequestBuilder.java:429)
		at com.ibm.cloud.cloudant.v1.Cloudant.putDocument(Cloudant.java:1552)
```

Tests are disabled until a newer version of `java-sdk-core` is available. Here is a [ticket](https://github.com/IBM/java-sdk-core/issues/233) on the project.

--- a/extensions/couchdb/runtime/pom.xml
+++ b/extensions/couchdb/runtime/pom.xml
@@ -46,6 +46,16 @@
         <dependency>
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-couchdb</artifactId>
+            <exclusions>
+                <exclusion>
+                    <artifactId>sdk-core</artifactId>
+                    <groupId>com.ibm.cloud</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <artifactId>sdk-core</artifactId>
+            <groupId>com.ibm.cloud</groupId>
         </dependency>
     </dependencies>
 

--- a/integration-tests/couchdb/pom.xml
+++ b/integration-tests/couchdb/pom.xml
@@ -80,7 +80,6 @@
 
 
     <profiles>
-        <!-- see https://github.com/apache/camel-quarkus/issues/7269
         <profile>
             <id>native</id>
             <activation>
@@ -108,7 +107,6 @@
                 </plugins>
             </build>
         </profile>
-        -->
         <profile>
             <id>virtualDependencies</id>
             <activation>

--- a/integration-tests/couchdb/src/test/java/org/apache/camel/quarkus/component/couchdb/it/CouchdbTest.java
+++ b/integration-tests/couchdb/src/test/java/org/apache/camel/quarkus/component/couchdb/it/CouchdbTest.java
@@ -22,7 +22,6 @@ import io.quarkus.test.common.QuarkusTestResource;
 import io.quarkus.test.junit.QuarkusTest;
 import io.restassured.RestAssured;
 import io.restassured.http.ContentType;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import static org.awaitility.Awaitility.await;
@@ -32,7 +31,6 @@ import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 @QuarkusTest
 @QuarkusTestResource(CouchdbTestResource.class)
-@Disabled("https://github.com/apache/camel-quarkus/issues/7269")
 class CouchdbTest {
 
     @Test

--- a/pom.xml
+++ b/pom.xml
@@ -117,6 +117,7 @@
         <hapi-base.version>${hapi-base-version}</hapi-base.version>
         <hapi-fhir.version>${hapi-fhir-version}</hapi-fhir.version>
         <hapi-fhir-core.version>6.4.0</hapi-fhir-core.version><!-- @sync ca.uhn.hapi.fhir:hapi-fhir:${hapi-fhir.version} prop:fhir_core_version -->
+        <ibm.java-sdk-core.version>9.23.1</ibm.java-sdk-core.version><!-- Required because of https://github.com/apache/camel-quarkus/issues/7269 -->
         <ibm.mq.client.version>9.4.2.0</ibm.mq.client.version>
         <icu4j.version>${icu4j-version}</icu4j.version>
         <influxdb.version>${influx-java-driver-version}</influxdb.version>

--- a/poms/bom/pom.xml
+++ b/poms/bom/pom.xml
@@ -6925,6 +6925,11 @@
                 <version>${re2j.version}</version>
             </dependency>
             <dependency>
+                <groupId>com.ibm.cloud</groupId>
+                <artifactId>sdk-core</artifactId>
+                <version>${ibm.java-sdk-core.version}</version>
+            </dependency>
+            <dependency>
                 <groupId>com.ibm.icu</groupId>
                 <artifactId>icu4j</artifactId>
                 <version>${icu4j.version}</version>

--- a/poms/bom/src/main/generated/flattened-full-pom.xml
+++ b/poms/bom/src/main/generated/flattened-full-pom.xml
@@ -6833,6 +6833,11 @@
         <version>1.7</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
       </dependency>
       <dependency>
+        <groupId>com.ibm.cloud</groupId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
+        <artifactId>sdk-core</artifactId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
+        <version>9.23.1</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
+      </dependency>
+      <dependency>
         <groupId>com.ibm.icu</groupId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
         <artifactId>icu4j</artifactId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
         <version>77.1</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->

--- a/poms/bom/src/main/generated/flattened-reduced-pom.xml
+++ b/poms/bom/src/main/generated/flattened-reduced-pom.xml
@@ -6823,6 +6823,11 @@
         <version>1.7</version>
       </dependency>
       <dependency>
+        <groupId>com.ibm.cloud</groupId>
+        <artifactId>sdk-core</artifactId>
+        <version>9.23.1</version>
+      </dependency>
+      <dependency>
         <groupId>com.ibm.icu</groupId>
         <artifactId>icu4j</artifactId>
         <version>77.1</version>

--- a/poms/bom/src/main/generated/flattened-reduced-verbose-pom.xml
+++ b/poms/bom/src/main/generated/flattened-reduced-verbose-pom.xml
@@ -6823,6 +6823,11 @@
         <version>1.7</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
       </dependency>
       <dependency>
+        <groupId>com.ibm.cloud</groupId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
+        <artifactId>sdk-core</artifactId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
+        <version>9.23.1</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
+      </dependency>
+      <dependency>
         <groupId>com.ibm.icu</groupId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
         <artifactId>icu4j</artifactId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
         <version>77.1</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->

